api/app.go

```diff
@@ -34,9 +35,15 @@ func NewApp(
 }
 
 func (a *App) configureApp() {
+	a.Config.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	a.Config.AutomaticEnv()
```

Comment:
These lines setting the configuration should be done before passing the config to the api, somewhere in cmd.

Also, you should set a prefix:
`config.SetEnvPrefix(...)`

Is it intentional that the application doesn't read any configuration from file?